src/components/ebay-textbox/index.js

```diff
@@ -0,0 +1,31 @@
+const markoWidgets = require('marko-widgets');
+const processHtmlAttributes = require('../../common/html-attributes');
+const template = require('./template.marko');
+
+function getInitialState(input) {
+    const classes = ['textbox__control'];
+    if (input.fluid) {
+        classes.push('textbox__control--fluid');
+    }
+    return {
+        classes,
+        htmlAttributes: processHtmlAttributes(input),
+        rootClass: input.class || '',
```

Comment:
For this, we usually use the pattern `['textboxt', input.class]`. No need to worry about falsey values, since Marko will handle that.